main.go

```diff
@@ -27,17 +26,73 @@ type envConfig struct {
 	Env string `envconfig:"ENV" default:"local"`
 	// URL of the Keptn configuration service (this is where we can fetch files from the config repo)
 	ConfigurationServiceUrl string `envconfig:"CONFIGURATION_SERVICE" default:""`
-	// URL of the Keptn event broker (this is where this service sends cloudevents to)
-	EventBrokerUrl string `envconfig:"EVENTBROKER" default:""`
```

Comment:
If we remove the option to configure the EVENTBROKER, we lose the possibility to use mock HTTP servers in the unit tests to check if the expected events are sent to the event broker. I suggest to keep the env var, but set it to `http://localhost:8081/event` (which is the event forwarding URL on the distributor) instead

Reply:
I'll add it again, but we need to discuss if we can run our unit tests in another way without relying on setting the environment variable.

Reply:
Actually, I have to raise a discussion, I can not just add it again.

Readding the environment variable might break existing services that upgrade to the new code, as they still have the old EVENTBROKER URL set (e.g., `http://eventbroker`).

https://github.com/keptn-sandbox/monaco-service/blob/6c8978ea987393de3f648f0292933b85a5e64076/deploy/service.yaml#L62-L63

https://github.com/keptn-sandbox/litmus-service/blob/7ba5db5d8d8b2d94adfff2bf3fa3864c39f8b5ac/deploy/service.yaml#L64-L65

https://github.com/keptn-sandbox/grafana-service/blob/4e41b905030be40f4b9656571d5411995923313f/deploy/service.yaml#L24-L25

Also, `go-utils` does not properly detect if we are sending a cloud-event to a non existign URL - see  https://github.com/keptn/keptn/issues/2919.

Reply:
Another way of testing woud be to have some sort of `EventSender` interface as a property within the KeptnHandler in the go utils, and have the possibility to inject it to the Keptn handler (per default it will be the implementation which sends the events to the distributor). In the unit tests, we would then have the possibility to inject a mock implementation where we can check for Events sent by the service. This way, we would not be reliant on the environment variable anymore. So I guess we can remove it here, and refactor the go-utils as suggested

Reply:
Thanks for your feedback. I agree, we should tackle this in go-utils.

Anyway, I just looked, and we already have the ability to "kind of" mock this:
https://github.com/keptn/go-utils/blob/86a445dfcb86450ff0c6e5b7216f8413a4c35e96/pkg/lib/v0_2_0/events.go#L71-L74

It looks like that if we set `keptnOptions.UseLocalFileSystem` to `true`, we will not send the CloudEvent at all, but print it. We are setting this in our `initializeTestObjects` within our tests.
Here we also could easily overwrite `keptnOptions.eventBrokerUrl` to a mock http endpoint, so I guess right now we don't need to change anything, it will work as it is.